sdk/export/metric/metric.go

```diff
@@ -418,12 +407,6 @@ func CumulativeExportKindSelector() ExportKindSelector {
 	return ConstantExportKindSelector(CumulativeExportKind)
 }
 
-// DeltaExportKindSelector returns an ExportKindSelector that
```

Comment:
Why dropping it? The `DeltaExportKind` still exists, so there could be a 3rd party Processor that actually supports converting precomputed sums to delta. Dropping `DeltaExportKindSelector` would force the existing users to change that to `ConstantExportKindSelector(DeltaExportKind)`, no?

Reply:
Ah, that's a terminology problem that we've marked for fixing in my derelict spec PR, see here: https://github.com/open-telemetry/opentelemetry-specification/pull/1159#discussion_r520960248. `ExportKind` should become `AggregationTemporality` but I would like to do that in a follow-on PR since it touches all the same files.

Reply:
I realized that the confusion I created has a lot to do with the misleading name `ExportKind`, so I added it in this PR. The new diff is pretty large, but it's almost entirely simple text substitution see 74c45ea.